packages/svelte/src/index.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @hawk.so/javascript-svelte
+ * Svelte/SvelteKit integration for Hawk.so JavaScript error tracker
+ */
+
+// Just verification that main sources may be used in current workspace
+// TODO: should be replaced with actual code during further implementation
```

Comment:
Are these docs still actual?

Reply:
I thought there would be something else here to make Hawk work in svelte, but it looks like just `HawkCatcher` is enough, so I think these comments are no longer needed.